src/content-helper/common/base-wordpress-provider.tsx

```
@@ -0,0 +1,316 @@
+		const hydratedPosts = posts.map( ( post ) => {
+			// Initialize empty arrays for the data we want to extract.
+			let categories: Taxonomy[] = [];
+			let tags: Taxonomy[] = [];
+			let thumbnail = '';
+
+			// Extract author data from _embedded.
+			const author = post._embedded?.author ? post._embedded.author[ 0 ] : null;
+
+			// Extract categories and tags data from _embedded.
+			// The first element in the array is categories, the second is tags.
+			if ( post._embedded && post._embedded[ 'wp:term' ] ) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)